orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -241,39 +241,42 @@ def forward(self):
         All other methods are internal"""
         pass
 
-    def _build_graph(self):
+    def _build_graph(self, training):
```

Comment:
do we need this extra param? Can we use self._export_mode?

Reply:
Can we move logic specific to training and eval to their respective graph managers (either inside `TrainingManager` or `InferenceManager` instead of having conditionals in the base class?

Reply:
Refactored again.